engines/instance_verification/lib/instance_verification/engine.rb

```diff
@@ -1,4 +1,15 @@
+require 'net/http'
+
+SUBSCRIPTIONS_PRODUCTS_URL = 'https://scc.suse.com/connect/subscriptions/products'.freeze
+
 module InstanceVerification
+  extend ::Net::HTTPHeader
+
+  def self.verification_basic_encode(login, password)
```

Comment:
Is this method used from anywhere? I wasn't able to find a usage.

Reply:
Yes, forgot to remove during the cleaning of the engine